### PR TITLE
[TwigBridge] Removed duplicated code from TwigRenderer

### PR DESCRIPTION
**Pull request by hason**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets |  |
| License | MIT |
| Doc PR |  |

See https://github.com/symfony/symfony/blob/2.7/src/Symfony/Component/Form/FormRenderer.php#L65-L69

### CODE REVIEW

File: src/Symfony/Bridge/Twig/composer.json [hunk 1]
```diff
@@ -17,7 +17,6 @@
     ],
     "require": {
         "php": ">=5.3.9",
-        "symfony/security-csrf": "~2.6|~3.0.0",
```
Comment (stof) [inline src/Symfony/Bridge/Twig/composer.json]: remocing the requirement looks weird to me

Comment (hason) [inline src/Symfony/Bridge/Twig/composer.json]: This is an optional dependency optionally required in the Form component.

Comment (WouterJ) [inline src/Symfony/Bridge/Twig/composer.json]: it should be moved to the `suggest` section then

Comment (WouterJ) [inline src/Symfony/Bridge/Twig/composer.json]: and probably `require-dev` to be able to test it

Comment (hason) [inline src/Symfony/Bridge/Twig/composer.json]: In ``require-dev`` section is  ``symfony/security`` which replaces ``symfony/security-csrf``. The ``symfony/security-csrf`` is optional dependency for Form component, see https://packagist.org/packages/symfony/form.

**Comment by fabpot** [comment]
Thank you @hason.

**Pull request closed without merging**